bigchaindb/tendermint/lib.py

```diff
@@ -90,7 +90,7 @@ def _process_status_code(self, status_code, failure_msg):
 
     def get_latest_block_height_from_tendermint(self):
         r = requests.get(ENDPOINT + 'status')
-        return r.json()['result']['latest_block_height']
+        return r.json()['result']['sync_info']['latest_block_height']
```

Comment:
Does `'sync_info'` paramter hold the syncing state with network or the abci proxy app?
On my FDN node I see the following information,

`/abci_info`
```json
{
  "jsonrpc": "2.0",
  "id": "",
  "result": {
    "response": {
      "last_block_height": 135,
      "last_block_app_hash": "NmJjM2FiZjY0M2JhMGQwYjY0ODIwODk5N2UwZGVkYTY4MzYwMjFmODI3YTRlNmQyNTlmMThmNTVkMTM3MzgyOQ=="
    }
  }
}
```

`/status`

```json
    "sync_info": {
      "latest_block_hash": "0C37B9B3CCD6B243096B971F329342993DD4C29E",
      "latest_app_hash": "36626333616266363433626130643062363438323038393937653064656461363833363032316638323761346536643235396631386635356431333733383239",
      "latest_block_height": 136,
      "latest_block_time": "2018-05-30T14:13:49.76567914Z",
      "syncing": false
    }
```

Reply:
Good point, I'm not sure if we should use `/abci_info` or `status`, will check now. But we cannot use `bigchain.get_latest_block()` because it checks our local MongoDB, that does not contain empty blocks.

Reply:
OK I'll go for `/status` :100: 

![image](https://user-images.githubusercontent.com/134680/40786805-4634188c-64ec-11e8-8b5e-7b36a653d605.png)